community/planning/cylinder_jwt.md

```diff
@@ -0,0 +1,227 @@
+# Cylinder JWT
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Summary
+[summary]: #summary
+
+This proposal adds a JSON Web Token (JWT) module to the Cylinder library.
+
+## Motivation
+[motivation]: #motivation
+
+Implementing a JWT feature in Cylinder allows for the use of its own signer
+implementations to provide signed tokens. This provides Cylinder users to make
+use of the same keys as an identity for use in both HTTP authentication tokens
+(via JWT) as well as the current comment use case of transaction signers.
+
+Existing JWT libraries do not allow for the flexibility to use custom signers.
+
+##Guide-level explanation
```

Comment:
Missing space 
`## Guide-level explanation`

This causes the formatting to get screwed

Reply:
fixed.